### PR TITLE
Remove large live preview top border, consolidate large preview css

### PR DESCRIPTION
**Pull request by arunshenoy99**
Also makes the color and typography large preview backgrounds consistent with preview and header menu.

### CODE REVIEW

File: src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss [hunk 1]
```diff
@@ -13,13 +13,20 @@ $main-color-light-rgb: var(--nfd-onboarding-white-rgb);
...
 		&-large {
 			width: 70%;
-			height: 70%;
+			min-height: 90vh;
 			overflow: hidden;
-			max-width: 925px;
-			max-height: 925px;
+			position: relative;
+			align-items: center;
+			border: 1px solid #e3dfdf;
```
Comment (Yashita101) [inline src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss:23]: Could we add the gray shade `#e3dfdf` to branding variables too?

Comment (arunshenoy99) [reply src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss:23]: added this to the hardcode removal task

Comment (Yashita101) [reply src/OnboardingSPA/components/LivePreview/BlockPreview/stylesheet.scss:23]: As discussed, these values shall be consolidated altogether as brand variables separately.

**Review by Yashita101 — APPROVED** [review]

**Comment by Yashita101** [comment]
Performed QA for the story, and all flows seem to work fine. 
This story is good to be merged.

**Comment by arunshenoy99** [comment]
> Performed QA for the story, and all flows seem to work fine. This story is good to be merged.

Thanks! @Yashita101

**Pull request merged**